include/FNA3D.h

```diff
@@ -401,6 +401,34 @@ typedef struct FNA3D_PresentationParameters
 	FNA3D_RenderTargetUsage renderTargetUsage;
 } FNA3D_PresentationParameters;
 
+typedef struct FNA3D_RenderTargetBinding
+{
+	/* FNA3D */
+	#define RENDERTARGET_TYPE_2D 0
+	#define RENDERTARGET_TYPE_CUBE 1
+	uint8_t type;
+
+	/* Texture */
+	FNA3D_SurfaceFormat format;
+	int32_t levelCount;
+	uint32_t textureHandle;
+
+	/* IRenderTarget */
+	int32_t width;
+	int32_t height;
+	FNA3D_RenderTargetUsage renderTargetUsage;
+	uint32_t depthStencilBufferHandle;
+	uint32_t colorBufferHandle;
+	FNA3D_DepthFormat depthStencilFormat;
+	int32_t multiSampleCount;
+
+	/* RenderTarget2D/Cube */
+	uint8_t isContentLost;
```

Comment:
Do we need this...?

Reply:
Probably not, at least not until we have evidence that any device would ever lose a target in 2020... To that end, I would cut any variables in here that are not in active use, until we know for sure that they're needed.

This looks good to me, how does it look on the managed side?